lib/utils/kube.go

```diff
@@ -428,3 +352,85 @@ func GetNodes(clientset *kubernetes.Clientset) ([]corev1.Node, error) {
 
 	return nodes.Items, nil
 }
+
+func CopyTarIntoPod(podName string, containerName string, pathInPod string, localFilePath string, ns ...string) error {
```

Comment:
Isn't this the same function? why change the name then?

Reply:
changed to original